src/PipelinesMicroserviceCLI/Commands/PublishPipeline.php

```diff
@@ -12,53 +12,42 @@
 use Symfony\Component\Console\Question\ChoiceQuestion;
 use Symfony\Component\Console\Question\ConfirmationQuestion;
 
-class PublishPipeline extends CLICommand
+class PublishPipeline extends PipelineManagerAPICommand
 {
-    protected function configure()
-    {
-        $this
-        ->setName('pipeline:publish')
-        ->setDescription('Publish a pipeline')
-        ->addArgument(
-            'base_url',
-            InputArgument::REQUIRED,
-            'The location of the pipeline microservice'
-        );
-        
-    }
+    protected $commandName = 'pipeline:publish';
```

Comment:
It'd be more standard to still extend ```configure()```.  Any configuration file we need on an application level is not releated to the configuration method in the commands.  And so that app level configuration is best injected in the constructor.